content/webapp/pages/whats-on.tsx

```diff
@@ -208,13 +208,26 @@ const Header: FunctionComponent<HeaderProps> = ({
       <div className="container">
         <div className="grid">
           <div className={grid({ s: 12, m: 12, l: 12, xl: 12 })}>
-            <div className="flex flex--v-center flex--h-space-between flex--wrap">
+            <div
+              style={{
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                flexWrap: 'wrap',
```

Comment:
This one too, rule of thumb we're thinking 1-2 properties is alright but more than that should/could be a styled component

Reply:
It could even be a styled component based on the one for https://github.com/wellcomecollection/wellcomecollection.org/pull/9572/files#diff-7b8f403372c6e80d526c764f28f5bcfa392f1afa800f5a355a815e5bb9a83590R541